src/js/components/Calendar/StyledCalendar.js

```diff
@@ -40,9 +56,23 @@ const weeksContainerSizeStyle = (props) => {
 
   `;
 };
+
+const weeksContainerResponsiveSizeStyle = (props) => {
+  const breakpoint = props.theme.global.size[props.sizeProp];
+  return breakpointStyle(
+    { value: breakpoint },
+    `
+    height: auto;
+    aspect-ratio: 7/6;
```

Comment:
Remind me why 7/6? Could we add a comment for future reference?

Reply:
Answered via slack but replying here also for others benefit. 7 days per week, 6 weeks shown. Pre-existing code does 14.6% width (100%/7) and 6*daySize height.

Reply:
Good to know, agree adding a comment in the code would be helpful for future

Reply:
Done